### ISSUE TITLE
CHe does not store events that come from a remote instance

### ISSUE THREAD

**Issue by Alfiva** [universAAL/context#23]
_Originally Opened: @Alfiva (2011-05-17 12:17:06_)
_Originally Closed: 2011-05-17 12:23:03_

When interconnecting two instances of uAAL, and the CHe is deployed and started in the one that is the coordinator, it does not store events that are geneated in the other instance.

What is more, it seems that those events are not even forwarded at all in the context bus of the coordinator (the instance that received them, not generated them).

This does not happen in the mobile version of he CHe. Integration tests proved that it works when interconnecting a PC instance with an Android instance.

--

From: _this issue has been automatically imported from our old issue tracker_

**Comment by Alfiva** [comment]
_Original author: @Alfiva (2011-05-17 12:22:51)_

The problem was that once the CHe was started in the coordinator, all context events received from remote instances were parsed as the fake context events used in CHe ontology, because they have the same URI. Therefore they were discarded by the bus.

This has been fixed by simply having a different MY_URI for the fake context event class of CHe. In consequence it required to change a bit of the code of the CHe services.

I have carried out some qick tests and it appears that this change has not affected the operativeness of the CHe. It keeps working as before, and its clients don´t need to change anything.
